src/pages/room/[roomId].tsx

```diff
@@ -17,16 +19,69 @@ const Room: NextPage<RoomProps> = ({ room }) => {
       </Head>
 
       <main>
-        <h1 className="text-primary">{room.name}</h1>
-        <button>CREATE POST</button>
-        <div className="d-flex flex-column">
-          <Link href={`${room.id}/1`}>Thread 1</Link>
-          <Link href={`${room.id}/2`}>Thread 2</Link>
-          <Link href={`${room.id}/3`}>Thread 3</Link>
-          <Link href={`${room.id}/4`}>Thread 4</Link>
-          <Link href={`${room.id}/5`}>Thread 5</Link>
+        <div className="bg-dark vh-100">
+          <div className="d-flex justify-content-between p-2">
+            <h1 className="text-primary">{room.name}</h1>
+            <button className="text-secondary h3">Raumoptionen</button>
+            {/*Popup für Raumoptionen hinzufügen */}
+          </div>
+          <div className="p-2">
+            <button className="btn btn-primary">CREATE POST</button>
```

Comment:
```suggestion
            <Button>CREATE POST</Button>
```
Bitte zu Buttons von `react-bootstrap` wechseln (muss noch importiert werden)
- theoretisch kann man dann auch die `className` property weglassen, da `btn` von `react-bootstrap` hinzugefügt wird und `btn-primary` der default dort ist